test/tests/functions/baselinecorrection/tests

```diff
@@ -0,0 +1,77 @@
+[Tests]
+  issues = '#296'
+  design = functions/BaselineCorrection.md
+
+  [test_correction]
+    requirement = "The BaselineCorrection class shall accurately adjust a raw acceleration time history according to the method of using least squares polynomial fits."
```

Comment:
The BaselineCorrection class shall accurately adjust a raw acceleration time history according to the method of using least squares polynomial fits for